rest-api/src/test/scala/za/co/absa/enceladus/rest_api/services/DatasetServiceTest.scala

```diff
@@ -345,10 +346,10 @@ class DatasetServiceTest extends VersionedModelServiceTest[Dataset] with Matcher
     Mockito.when(modelRepository.getVersion("datasetA", 1)).thenReturn(Future.successful(Some(dataset)))
     Mockito.when(modelRepository.getLatestVersionValue("datasetA")).thenReturn(Future.successful(Some(1)))
 
-    val maybeDataset = await(service.update("datasetA",
+    val maybeDataset: Option[(Dataset, Validation)] = await(service.update("datasetA",
       DatasetFactory.getDummyDataset(name = "datasetA", description = Some("newdescr"))))
-    assertResult(Some("newdescr"))(maybeDataset.get.description)
-    assert(!maybeDataset.get.lockedWithDefault)
+    assertResult(Some("newdescr"))(maybeDataset.get._1.description)
```

Comment:
I added these changes manually. Maybe if @dk1844 can double check that the code matches his